conjur/controller/init_controller.py

```diff
@@ -159,6 +159,6 @@ def ensure_overwrite_file(cls, config_file):
         Method to handle user overwriting logic
         """
         force_overwrite = input(f"File {config_file} exists. "
-                                f"Overwrite? (Default=yes): ").strip()
+                                f"Overwrite? yes/no (Default: yes): ").strip()
```

Comment:
You missed `Trust this certificate? yes/no (Default=no)`. This should be Default: no